3.resistance-signature/results/singscore/UNIQUE_README.md

```diff
@@ -0,0 +1,6 @@
+UNIQUE_README.md
```

Comment:
Please use one-sentence-per-line syntax for all `.md` files.

I'll demonstrate an example below